QualityControl/public/layout/list/page.js

```diff
@@ -74,77 +74,100 @@ function createHeaderOfFolder(model, folder) {
+ * @returns {vnode} - A virtual DOM node representing the card group layout.
  */
-function table(model, layouts, searchBy) {
-  return [
-    h(
-      'table.table.table-sm',
-      [
-        h(
-          'thead',
-          h(
-            'tr',
-            [
-              h('th', h('.text-center', 'Official')),
-              h('th', 'Name'),
-              h('th', 'Owner'),
-              h('th', 'Description'),
-              h('th', h('.text-right', 'Actions')),
-            ],
-          ),
-        ),
-        h('tbody', rows(model, layouts, searchBy)),
-      ],
-    ),
-  ];
-}
-
-/**
- * Shows layouts as table lines
- * @param {Model} model - root model of the application
- * @param {RemoteData} layouts - list of layouts as remoteData object
- * @param {string} searchBy - string to search by in the list of layouts
- * @returns {vnode} - virtual node element
- */
-function rows(model, layouts, searchBy) {
+function layoutCards(model, layouts, searchBy) {
   return layouts.match({
     NotAsked: () => null,
-    Loading: () => h('', 'Loading...'),
-    Failure: () => h('tr', [h('td', 'Unable to retrieve this list of layouts')]),
+    Loading: () => h('div', 'Loading...'),
+    Failure: () => h('div', [h('div.alert.alert-danger', 'Unable to retrieve this list of layouts')]),
     Success: (list) => {
       if (!list || list.length <= 0) {
-        return h('tr', [h('td', 'No layouts found')]);
+        return h('div', [h('.cardGroupRow', 'No layouts found')]);
       }
-      return list.filter((item) => item.name.match(searchBy))
+      return h('.cardGroupRow', list.filter((item) => item.name.match(searchBy))
         .map((layout) => {
-          const key = `key${layout.name}`;
+          const { description, owner_name, name, id } = layout;
           const { isOfficial } = layout;
           const isMinimumGlobal = model.session.access.some((role) => isUserRoleSufficient(role, UserRole.GLOBAL));
-          return h('tr', { key: key }, [
-            h('td', {
-            }, isOfficial ? h('.primary.f4.text-center', [iconBadge(), ' ']) : ' '),
-            h('td.w-20', [
-              h('.flex-row.items-center', [
-                h('a', {
-                  href: `?page=layoutShow&layoutId=${layout.id}`,
-                  onclick: (e) => model.router.handleLinkEvent(e),
-                }, layout.name),
-              ]),
-            ]),
-            h('td.w-30', layout.owner_name),
-            h('td.w-30', layout.description ?? '-'),
-            h('td', h('.text-right', [
-              isMinimumGlobal && h('button.btn.btn-sm', {
-                onclick: () => model.layout.toggleOfficial(layout.id, !layout.isOfficial),
-              }, layout.isOfficial ? 'Un-official' : 'Official'),
-            ])),
+          const toggleOfficialFunction = (id) => model.layout.toggleOfficial(id);
+
+          return h('.p2.card', [
+            cardHeader(isOfficial, id, name, isMinimumGlobal, toggleOfficialFunction),
```

Comment:
I think this is cleaner
```suggestion
            cardHeader({ ...layout, isMinimumGlobal, toggleOfficialFunction })
```